src/test/resources/pages/instructorFeedbackSubmitPagePrivate.html

```diff
@@ -52,6 +52,7 @@
               </label>
               <div class="col-sm-10">
                 <p class="form-control-static">
+                  ${datetime.now} UTC+0800
```

Comment:
@HirdayGupta The `dateToCalendar` method in TimeHelper returns current date time when the parameter given is null, which in turns affect `convertLocalDateToUtc` in TimeHelper and `getEndTimeString` in FeedbackSessionAttributes. One possible way to solve the problem is to disable display of end time if it is a private session.